docs/architecture.md

```diff
@@ -16,6 +16,8 @@ The K8TRE Reference Implementation uses the default high- and low-level containe
 
 ### GitOps
 
+The K8TRE Reference Implementation uses ArgoCD installed on a management cluster to manage nearly all resources on the child cluster(s) it manages. Here "nearly all" means ArgoCD will not be responsible for creating/destroying workspaces. 
```

Comment:
I'd prefer that this statement was not redundant with the content in gitops.md

I'd suggest that the architecture only contains top-level stuff that doesn't belong in a spec doc.